README.md

```diff
@@ -245,7 +245,7 @@ These variables are the defaults of our roles, if you want to override the prope
 | provision_postgres | Run the postgres provisioning contained into postgres role, set to false if you provide an external postgres to the services | "true" |
 | provision_rabbitmq | Run the rabbitmq provisioning contained into rabbitmq role, set to false if you provide an external rabbitmq to the services | "true" |
 | provision_proxy | Run the nginx provisioning for exposing all the services, se to false if you don't want to expose the services or you have already in place a reverse proxy infrastructure | "true" |
-| provision_prometheus | Run the prometheus provisioning used by trento to store metrics send by agents | "true" |
+| provision_prometheus | Run the prometheus provisioning used by trento to store metrics send by agents | true when enable_charts is true or undefined |
```

Comment:
I think the default value should be `true`, The other conditions should be explained in the description.
The same goes for `enable_charts`